plugins/catalog-react/src/components/UserListPicker/UserListPicker.tsx

```diff
@@ -223,22 +230,19 @@ export const UserListPicker = ({
                   onClick={() => setSelectedUserFilter(item.id)}
                   selected={item.id === filters.user?.value}
                   className={classes.menuItem}
+                  disabled={filterCounts[item.id] === 0}
+                  data-testid={`user-picker-${item.id}`}
```

Comment:
Moved the testid up here to make it simpler to check whether the menu items are enabled in the DefaultCatalogPage tests. Hoping this doesn't break any other uses of this attribute - will see if CI is happy!